src/util/helpers.tsx

```diff
@@ -335,3 +336,8 @@ export const getDefaultStoragePool = (profile: LxdProfile) => {
     });
   return rootStorage ? rootStorage.pool : "";
 };
+
+export const base64EncodeTokenDetails = (data: LxdFineGrainedTlsIdentity) => {
```

Comment:
I think we should make this a bit more generic, i.e. call it `base64EncodeObject(data: Object)` so it can be reused in other contexts.